unity/Assets/Scripts/Unity/UnityMap.cs

```diff
@@ -104,6 +105,15 @@ public GameObject TileObjectOnPoint(Vector3 point)
         }
 
         public void OnTilePlaced(object sender, TileMapEventArgs<ITile> e)
+        {
+            ITile tile = e.Tile;
+            GameObject template = TileTemplate;
+            Material material;
+            material = materials[(int)e.Tile.Type.Type - 1];
+            StartCoroutine(PlaceTileDelayed(tile, template, material, CoordinateToPoint(tile.Coordinate), Quaternion.Euler(0, 0, 0), e.Tile.Type.Type));
+        }
+
+ /*       public void OnTilePlaced(object sender, TileMapEventArgs<ITile> e)
```

Comment:
If it's not used, delete it. You don't have to worry about keeping "old" stuff just to be able to go back to it. We can always look it up via git.